src/main/java/org/folio/verticle/consumers/ConsortiumDeleteEventsHandler.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.verticle.consumers;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.kafka.client.consumer.KafkaConsumerRecord;
+import io.vertx.kafka.client.producer.KafkaHeader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.event.service.UserTenantService;
+import org.folio.kafka.AsyncRecordHandler;
+import org.folio.kafka.KafkaHeaderUtils;
+import org.folio.rest.jaxrs.model.UserTenant;
+import org.folio.rest.utils.OkapiConnectionParams;
+
+import java.util.List;
+
+public class ConsortiumDeleteEventsHandler implements AsyncRecordHandler<String, String> {
+
+  private static final Logger logger = LogManager.getLogger(ConsortiumDeleteEventsHandler.class);
+
+  private final Vertx vertx;
+  private final UserTenantService userTenantService;
+
+  public ConsortiumDeleteEventsHandler(Vertx vertx) {
+    this.vertx = vertx;
+    this.userTenantService = new UserTenantService();
+  }
+
+  @Override
+  public Future<String> handle(KafkaConsumerRecord<String, String> record) {
+    Promise<String> result = Promise.promise();
+    List<KafkaHeader> kafkaHeaders = record.headers();
+    OkapiConnectionParams okapiConnectionParams = new OkapiConnectionParams(KafkaHeaderUtils.kafkaHeadersToMap(kafkaHeaders), vertx);
```

Comment:
okapiConnectionParams now not used, so should be deleted from both consumer created/consumer deleted handlers